From df18044981f386e3f503c88caf66551a2ada282c Mon Sep 17 00:00:00 2001
From: NeKit <nekit1000@gmail.com>
Date: Sun, 9 Feb 2020 19:33:58 +0100
Subject: [PATCH] (halium) disable keystore service

Disabled in Sailfish ports, but may be needed for fingerprint later on?

Change-Id: I28489bfa74653f07c7acf78a02e1b72c26d0f047
---
 keystore/keystore.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/keystore/keystore.rc b/keystore/keystore.rc
index 132039a..45c9571 100644
--- a/keystore/keystore.rc
+++ b/keystore/keystore.rc
@@ -1,4 +1,4 @@
-service keystore /system/bin/keystore /data/misc/keystore
+service keystore /system/bin/keystore_HYBRIS_DISABLED /data/misc/keystore
     class main
     user keystore
     group keystore drmrpc readproc log
-- 
2.25.0

